docs/system_overview/configuration.md

```diff
@@ -69,13 +82,20 @@ This file defines the behavior of the server Agent that answers individual reque
         // The amount of time a temporary password remains valid.
         "maximum_temporary_password_lifetime_in_seconds": 1000,
 
+        // (Optional)
+        // The number of seconds between runs of the CRON task which runs the which executes the
```

Comment:
"which runs the which"?

Reply:
Ah, that's why that was weird. Been staring at these too much.

Reply:
Fixed